### DISCUSSION TITLE
Plugin Request - Reduced Motion

### DISCUSSION THREAD

**Discussion by Snowearthh** [runelite/runelite#17898] [Ideas]
**Title:** Reduced Motion / Particle Effects
**Description:** If possible please reduce the screen shaking and visual effects at certain places as it causes me to get motion sickness
This would be a huge accessibility feature for a lot of people

**Additional Information:** Relevant in places like Olm transitional phases and final head phase where the screen starts shaking a ridiculous amount, TOA enrage phase with the screen stuttering and absurd amount of particle effects. Currently, I am not able to complete TOA due to the screen shake and particle effects as it causes me to get dizzy.

Thank you for your time.

**Comment by Snowearthh** [comment]
bump

**Comment by Nightfirecat** [comment]
I really suggest directing this request to Jagex instead, as it would be useful for accessibility changes to not require a specific third-party client to address.

**Reply by Snowearthh** [reply]
I have tried. Their response was rather quite disappointing.

Link: https://twitter.com/Snowearthh/status/1681046587999387648
